src/java.base/share/classes/java/lang/annotation/AnnotationFormatError.java

```diff
@@ -45,7 +49,7 @@ public class AnnotationFormatError extends Error {
      *
      * @param   message   the detail message.
      */
-    public AnnotationFormatError(String message) {
+    public AnnotationFormatError(@Nullable String message) {
```

Comment:
This was the one place that jumped out to me as possible controversial: The class documents that `cause` can be null, but it's silent on `message`. I think there's a lot of precedent for the idea that `message` can be null for exceptions in general. But I also doubt that anyone will ever care what we do here, so I'm happy to fold and leave it non-nullable if you think it's better to err on that side in the absence of docs.

Reply:
Using `@Nullable` is consistent with what we do elsewhere with `message`s, so I think it's good to add it. We should probably revisit that decision globally, as null is not the best message.

Reply:
Thanks. It feels a little like `WeakReference(@Nullable T t, ...)`: Nothing will blow up if you pass `null`, but it's suspicious that you would want to.